### PR TITLE
Add some embeds

### PR DESCRIPTION
**Pull request by ellell**
Type: Minor
Review: @danmakenoise @iefserge @kesla

### CODE REVIEW

File: lib/embeds/tumblr.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    iframe.__props__ = props;
+    const {url, onLoaded, onResize} = props;
+    const content = renderString(tree(renderEmbed({type: 'tumblr', url})));
+    const protocol = window.location.protocol === 'file:'
+      ? 'https:'
+      : '';
+
+    const script = `<script src="${protocol}//assets.tumblr.com/post.js"></script>
+      <script>
+      var interval = setInterval(function () {
```
Comment (danmakenoise) [inline lib/embeds/tumblr.js]: Nitpick, I prefer `setTimeout`, prevents runaway intervals.

**Comment by danmakenoise** [comment]
Also, travis is sad!

File: lib/embeds/tumblr.js (hunk 1 above)
Comment (kesla) [inline lib/embeds/tumblr.js]: wdym?

Comment (danmakenoise) [inline lib/embeds/tumblr.js]: @kesla, it's essentially the same, I just get in the habit of using `setTimeout` instead so you don't accidentally forget to clear intervals. looks like this:

```
function resizeTumblrEmbed() {
  var iframe = document.querySelector('.tumblr-embed');
  
  if (iframe.height) {
    resize();
  } else {
    setTimeout(resizeTumblrEmbed, 100);
  }
}
resizeTumblrEmbed();
```

Comment (ellell) [inline lib/embeds/tumblr.js]: Hm, not sure I agree. I find the interval to be easier to read here.

Comment (danmakenoise) [inline lib/embeds/tumblr.js]: @ellell no worries, it's just personal preference, no biggie :)

**Pull request merged**